.travis.yml

```diff
@@ -19,7 +20,7 @@ env:
     - MESOS_NATIVE_JAVA_LIBRARY=/usr/lib/libmesos.so
     - CLJ_HTTP_ASYNC_POOL_TEST_DURATION_MULTIPLIER=5
   matrix:
-    - COOK_EXECUTOR='1' TEST_DIR=executor DEPS_CMD='python3 setup.py install' TEST_CMD='python3 setup.py nosetests'
+    - COOK_EXECUTOR='1' TEST_DIR=executor DEPS_CMD='pip --version && pip install virtualenv && virtualenv --version && virtualenv py35 && ls -la py35/bin && source py35/bin/activate py35 && python setup.py install' TEST_CMD='python setup.py nosetests'
```

Comment:
Does the `py35` argument in `source py35/bin/activate py35` do something? I thought that the `activate` script didn't accept any options.

Reply:
You're right, removed the unnecessary argument.